app/javascript/packs/sol_prices/sol_price_component_template.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="card col-6">
+    <div class="card-content">
+      <div class="col-12">
+        sol price 
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import axios from 'axios'
+
+  export default {
+    data() {
+      return {
+        connection: null,
+        ftx_url: "https://ftx.com/markets/sol_usd"
+      }
+    },
+    methods: {
+      get_ftx_price() {
+        axios.get(this.ftx_url, {
```

Comment:
Following by documentation (https://docs.ftx.com/?javascript#websocket-api) I did establish a connection with FTX API, with success. I used the postman.

Did you try to use the following library?
```
const WebSocket = require("ws");
new WebSocket('wss://ftx.com/ws/);
```